bin/index.js

```diff
@@ -64,36 +64,12 @@ const askQuestions0 = () => {
 
 const askQuestions1 = () => {
   const questions = [
-    {
-      name: "_publicKey",
-      type: "input",
-      message: "Public Key?",
-      default: publicKey
-    },
-    {
-      name: "_secretKey",
-      type: "input",
-      message: "Secret Key?",
-      default: secretKey
-    },
     {
       name: "_cipJsonFilePath",
       type: "input",
       message: "CIP-72 json path/file-name?",
       default: cipJsonFilePath
     },
-    // {
-    //   name: "_cipFilePath",
-    //   type: "input",
-    //   message: "CIP-26 json output file name?",
-    //   default: cipFilePath
-    // },
-  ];
-  return inquirer.prompt(questions);
-};
-
-const askQuestions2 = () => {
-  const questions = [
     {
       name: "_metadataFilePath",
```

Comment:
I just realized that now we ended up with two questions for different metadatas...

Do we actually need this one? The path under this variable is used to generate a transaction metadata JSON file and then use it while building the TX. IMO we could remove this question completely along with all the related implementations (.env variables etc.) and just have a hardcoded name for the tx metadata json file and cleanup it at the end, same as we do with tx.draft.

Removing this question we avoid confusion with the next one about the metadata url.

Reply:
IMO, it is better we live the question so that users can override it and pass other paths not in the env to it.

Reply:
Hmmm. How about if the user wants to view the metadata that was generated and registered on-chain?

Personally, I favour keeping this question but wording it in a way that is less confusing.

Reply:
For the record: We had a discussion and agreed to remove the question, have the path hardcoded but don't remove the file and log the path for the user.

Reply:
Question removed here. 616a9de